Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -139,12 +211,73 @@ for temporary files of extensions and TYPO3 components.
 Directory for temporary files that should be public available
 (e.g. generated images).
 
-
 .. _directory-public-typo3temp-var:
 
-:file:`public/typo3temp/var/` or :file:`var/labels`
-"""""""""""""""""""""""""""""""""""""""""""""""""""
+:file:`public/typo3temp/var/`
+"""""""""""""""""""""""""""""
+
+Legacy installations only. Directory for temporary files that contains private
+files (e.g. cached Fluid templates) and should not be publicly available.
+See also :ref:`Environment-configuring-paths` for a more detailed description.
+
+.. _directory-var:
+
+:file:`var/`
+------------
 
-Directory for temporary files that contains private files (e.g. cached
-Fluid templates) and should not be publicly available.
+Composer installations only. Directory for temporary files that contains private files (e.g.
+cache and logs files) and should not be publicly available.
 See also :ref:`Environment-configuring-paths` for a more detailed description.
+
+.. _directory-var-cache:
+
+:file:`var/cache/`
+~~~~~~~~~~~~~~~~~~~
+
+Composer installations only. This directory contains internal files needed
+for the cache.
+
+.. _directory-var-labels:
+
+:file:`var/labels/`
+~~~~~~~~~~~~~~~~~~~
+
+Composer installations only.  The directory :file:`var/labels/` is for extension
```

Comment:
```suggestion
Composer-based installations only.  The directory :file:`var/labels/` is for extension
```

"... is for localizations. It contains all downloaded translation files." -> maybe better: "... contains all downloaded translations files." Just omit the "is for" part.